### PR TITLE
fix Governance.getProposalStage

### PR DESCRIPTION
**Pull request by AlexBHarley**
### Description

Running this command on Baklava is broken right now. Dug in a little and found a `this` binding issue that seems to have been known about. The PR mentioned in the comment has been merged.

### Other changes

N/A

### Tested

- `cli governance:list --node https://forno.celo.org`
- `cli governance:list --node https://alfajores-forno.celo-testnet.org`
- `cli governance:list --node https://baklava-forno.celo-testnet.org`

### Related issues

N/A

### Backwards compatibility

N/A

### CODE REVIEW

**Comment by yorhodes** [comment]
I thought this was the issue
```
diff --git a/packages/cli/src/commands/governance/list.ts b/packages/cli/src/commands/governance/list.ts
index 9453d2af8..242abda6b 100644
--- a/packages/cli/src/commands/governance/list.ts
+++ b/packages/cli/src/commands/governance/list.ts
@@ -39,7 +39,7 @@ export default class List extends BaseCommand {
     const dequeue = await governance.getDequeue(true)
     const expiredDequeueMap = await concurrentMap(5, dequeue, governance.isDequeuedProposalExpired)
     const unexpiredDequeue = dequeue.filter((_, idx) => !expiredDequeueMap[idx])
-    const stages = await concurrentMap(5, unexpiredDequeue, governance.getProposalStage)
+    const stages = await concurrentMap(5, unexpiredDequeue, (id) => governance.getProposalStage(id))
     const proposals = zip((proposalID, stage) => ({ proposalID, stage }), unexpiredDequeue, stages)

     console.log(chalk.blue.bold('Dequeued Proposals:'))
```

**Comment by AlexBHarley** [comment]
@yorhodes that's the same thing right, I'm just deciding to fix the issue at the function level itself rather than at the call site.

My understanding is that non arrow functions allow their `this` context to be bound differently.

```
function getProposalStage () {
  // "this" is undefined or bound specifically in concurrentMap
}

const stages = await concurrentMap(5, unexpiredDequeue, governance.getProposalStage)
```

```
getProposalStage = () => {
  // "this" refers to its parent class
}

const stages = await concurrentMap(5, unexpiredDequeue, governance.getProposalStage)
```

Converting the function to an arrow function seems like the more long term fix.

**Comment by AlexBHarley** [comment]
Friendly ping on this @yorhodes. I think it's pretty safe to go in

File: packages/sdk/contractkit/src/wrappers/Governance.ts [hunk 1]
```diff
@@ -387,9 +387,7 @@ export class GovernanceWrapper extends BaseWrapper<Governance> {
   getApproverMultisig = () =>
     this.getApprover().then((address) => this.kit.contracts.getMultiSig(address))
 
-  async getProposalStage(proposalID: BigNumber.Value): Promise<ProposalStage> {
-    // NOTE: `methods.getProposalStage` is broken for case where proposal is queued and expired
-    // can be reverted once https://github.com/celo-org/celo-monorepo/pull/6502 is released
```
Comment (yorhodes) [inline packages/sdk/contractkit/src/wrappers/Governance.ts:392]: this hasn't been released, as confirmed by

```
./bin/run network:contracts --node https://forno.celo.org --filter Contract=Governance --columns Version,Contract
Version Contract
1.2.0.2 Governance
```

**Review by yorhodes — APPROVED** [review]

**Pull request merged**